website/docs/MigrationV12.md

```diff
@@ -0,0 +1,28 @@
+---
+id: migration-v12
+title: Migration to 12.0
+---
+
+Migration to React Native Testing Library version 12 from version 11.x should be a relatively easy task due small amount of breaking changes.
+
+# Breaking changes
+
+1. `*ByRole` queries now return only accessibility elements, either explicitly marked with `accessible` prop or implicit ones where this status is derrived from component type itself (e.g `Text`, `TextInput`, `Switch`).
+
+You may need to adjust relevant components under test to make sure they pass `isAccessibilityElement` check.
+
+2. `*ByText`, `*ByDisplayValue`, `*ByPlaceholderText` queries now return host components, as all other queries.
```

Comment:
add comparison pls

Reply:
Not sure what do you mean by adding comparison here. I guess that in most cases this will not affect users tests, so there is nothing to show here.

Reply:
Maybe that `returnedValue.type` is now always a string instead of a function for example?

Reply:
Yes, this is true, however we never encouraged users to inspect `instance.type`, as this is largely an implementation detail for the users.

Reply:
We never did, but what's possible, is gonna be used eventually. https://www.hyrumslaw.com